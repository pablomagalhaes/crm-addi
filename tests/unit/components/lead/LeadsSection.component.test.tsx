import React from 'react';
import { shallow } from 'enzyme';
import 'jest-styled-components';
import LeadsSection from '@/components/lead/LeadsSection.component';

jest.mock('@/io/redux/lead/useLeads', () =>
  jest.fn(() => ({
    leads: [],
    getLeads: jest.fn(),
  }))
);

describe('<LeadsSection />', () => {
  it('should match snapchat', () => {
    const wrapper = shallow(<LeadsSection />);
    expect(wrapper).toMatchSnapshot();
  });
});
