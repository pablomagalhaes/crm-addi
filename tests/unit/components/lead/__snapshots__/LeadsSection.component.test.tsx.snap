// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<LeadsSection /> should match snapchat 1`] = `
<Section
  title="Leads"
>
  <Leads
    leads={Array []}
  />
</Section>
`;
